### PR TITLE
#9346 Add S3 URLS for 151/131/111

### PR DESCRIPTION
**Pull request by dawn-minion**
Fixes Proemion/ts2-psp-merge#9346

### CODE REVIEW

**Comment by dawn-minion** [comment]
@unicolet Ping, since I can't assign you directly here

File: bin/install [hunk 1]
```diff
@@ -60,13 +60,16 @@ get_java() {
 	local base=http://download.oracle.com/otn-pub/java/jdk/
 	local oracle_version=$(echo $version | sed -e 's/\./u/g')
 
+	local s3base=https://s3-eu-west-1.amazonaws.com/repo.proemion.com
+
 	case $version in
 		9.0.1) url=$base$version+11/jdk-$version'_'$(get_current_variant $variant) ;;
 		8.151) url=$base$oracle_version-b12/e758a0de34e24606bca991d704f6dcbf/jdk-$oracle_version-$variant ;;
```
Comment (unicolet) [inline bin/install]: @dawn-minion let's upload 151 to s3 and use that here

Comment (dawn-minion) [reply bin/install]: @unicolet Will do

Comment (unicolet) [reply bin/install]: @dawn-minion thanks :-) We can leave 131 pointing to s3 since it's already there

**Comment by dawn-minion** [comment]
@unicolet Bumped PR with 151 version, and added it to the S3 bucket

**Review by unicolet — APPROVED** [review]
@dawn-minion nice! Can I merge?

**Comment by dawn-minion** [comment]
@unicolet Seems to work great locally, so, I think it's safe to now

**Pull request merged**